### PR TITLE
Authentication functionality

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Login and signup Functionality

login

### CODE REVIEW

File: src/pages/Login/Login.jsx [hunk 1]
```diff
@@ -1,7 +1,71 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import axios from "axios";
+import React, { useState } from "react";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { Link, useNavigate } from "react-router-dom";
 import "./Login.css";
+import { useAuth } from "../../contexts/auth-context";
```
Comment (yashpatel1797) [inline src/pages/Login/Login.jsx:7]: create index.js for import.

**Review by yashpatel1797 — APPROVED** [review]
code looks clean,
the file structure is well maintained.

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import "./App.css";
-
 import { Footer } from "./components/Footer/Footer";
 import { NavBar } from "./components/NavBar/NavBar";
 import { RoutersPath } from "./Routers/RoutersPath";
```
Comment (devdeadviz) [inline src/App.js]: You can use an `index.js` file and add all your exports here and you can simply import all your component from a single file.

**Review by devdeadviz — APPROVED** [review]
Code is clean and great folder structure.
Just some neat picks are there, have a look at it.

**Review by devdeadviz — APPROVED** [review]
Good to go, you have followed all the best approaches,

**Pull request merged**